### PR TITLE
Create warning and error list for third-party JS libraries

### LINKED ISSUE #598: Create a warning list for third party libraries
We need to warn reviewers of potentially dangerous or forbidden third party libraries.

My proposal would be to create a list of third party libraries we always warn about, similar to what we already have with dispensary for whitelisting.

### PR DESCRIPTION
**Pull request by tofumatt**
Fix #598

This won't work until https://github.com/mozilla/dispensary/pull/142 is merged.

### CODE REVIEW

**Comment by tofumatt** [comment]
r?

**Comment by wagnerand** [comment]
@tofumatt could you update this list with the recent angular additions?

**Comment by wagnerand** [comment]
We'd also need to get https://github.com/mozilla/addons-linter/blob/master/docs/third-party-libraries.md up and running before this lands on prod.

File: src/linter.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import { terminalWidth } from 'cli';
 import * as constants from 'const';
 import { ARCH_DEFAULT, ARCH_JETPACK, CHROME_MANIFEST, INSTALL_RDF,
          MANIFEST_JSON } from 'const';
+import { BANNED_LIBRARIES, UNADVISED_LIBRARIES } from 'libraries';
```
Comment (muffinresearch) [inline src/linter.js]: `UNADVISED` reads weird to me, but that's probably more because it's not commonly used.

**Comment by muffinresearch** [comment]
lgtm although there's a tiny bit of missing coverage.

r+wc (code)

File: src/linter.js (hunk 1 above)
Comment (tofumatt) [inline src/linter.js]: Yeah, I didn't know what to call it. They're like... "warning" or "caution"... I couldn't think of a better word 😞

**Comment by tofumatt** [comment]
We have one uncovered line, but that was the same as before, odd.

**Pull request merged**